### DISCUSSION TITLE
redirect from http/port(8080) to https/port(***)

### DISCUSSION THREAD

**Discussion by kassemchakaroun** [keycloak/keycloak#25773] [Q&A]
hello
hope you are all doing well!!
i want to redirect my keycloack service from http/port(8080) to https/port(4443)
can we do it in keycloack conf or we need a web server ?

**Comment by danielFesenmeyer** [comment]
Keycloak supports both approaches.
When you just want to redirect (and terminate TLS with a reverse proxy), you can configure the hostname as described here: https://www.keycloak.org/server/hostname
If you want Keycloak to handle TLS, you can configure that as described here: https://www.keycloak.org/server/enabletls
